### DISCUSSION TITLE
Start automatically on linux server

### DISCUSSION THREAD

**Discussion by jamespdaily** [itsToggle/plex_debrid#345] [Q&A]
Is there a way to start the rclone and plex_debrid services automatically after a reboot?

**Comment by ramphex** [comment]
also interested

**Comment by GSKang94** [comment]
I use this script on my rasp. pi. "PLEX" is my root folder for all plex-debrid stuff.
`touch plexScript.sh`

```bash
#!/bin/bash

# Navigate to /PLEX folder
cd /PLEX || exit 1

# Function to start screen session and wait for it to be ready
start_screen_session() {
    local session_name="$1"
    local command="$2"

    screen -S "$session_name" -d -m bash -c "$command"
    sleep 2

    while ! screen -list | grep -q "$session_name"; do
        sleep 1
    done
}

# Start rclone screen session
start_screen_session "rclone" "sudo ./rclone-linux-arm64 mount realdebrid: torrents --dir-cache-time 10s --allow-other"

# Wait for a moment to ensure the rclone screen session has started
sleep 5

# Start plex_debrid screen session
start_screen_session "plex_debrid" "sudo python ./plex_debrid/main.py"
```

To make this script auto-run on boot, add it to cronjob
`sudo chmod +x plexScript.sh`
`sudo crontab -e`
Add this line
`@reboot /path/to/your/plexScript.sh`

Note - You need to disable UI in plex-debrid settings to make this script work.

**Reply by niblem85** [reply]
This is really great for Pi users!  I am personally interested in the exact same thing, but for Windows 10. Wondering if anyone uses a similar script written for Windows?

**Reply by AndyZaa** [reply]
https://github.com/itsToggle/plex_debrid/discussions/525

**Comment by vdias** [comment]
Why do you guys recommend the use of screen for the mount part? Why not create a simple service to mount everything without using screen?
